app/controllers/supplejack_api/stories/moderations_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
         render json: {
           sets: user_sets.map { |user_set| StoriesModerationSerializer.new(user_set) },
           total_filtered: user_sets.count,
-          total: UserSet.public_sets_count,
+          total: UserSet.public_not_favourites.count,
```

Comment:
This is a confusingly named scope haha

Reply:
Talking about `public_sets_count` or `public_not_favourites` ?